src/dotnet/Program.cs

```diff
@@ -178,6 +182,9 @@ private static void ConfigureDotNetForFirstTimeUse()
                             environmentProvider);
 
                         dotnetConfigurer.Configure();
+
+                        if (!File.Exists(Telemetry.TelemetrySentinel))
```

Comment:
By default the path is `$HOME/.nuget/packages` which is user local.

https://github.com/dotnet/cli/blob/18c8a2724ad066a6999540f5015e1a0ed89b94fc/src/Microsoft.DotNet.ProjectModel/Resolution/PackageDependencyProvider.cs#L159

If global.json or NUGET_PACKAGES is specified then the user has taken some explicit measure to override this.